docs/tethys_sdk/url_maps.rst

```diff
@@ -107,7 +107,7 @@ The example below adds a column layout containing a slider and a plot. A callbac
                 data['y'] = [0, 10, 20, 30]
             else:
                 data['y'] = [i * new for i in [0, 10, 20, 30]]
-            source.data = ColumnDataSource(data=data).data
+            source.data = dict(ColumnDataSource(data=data).data)
```

Comment:
@msouff is this the recommended approach in the new version of Panel? What is the reason for casting this to a dictionary?

Reply:
Yes. This is the recommended now. The data parameter of ColumDataSource only takes plain Python dicts now. See the error below and the recommended change.

ValueError('\nColumnDataSource.data properties may only be set from plain Python dicts,\nnot other ColumnDataSource.data values.\n\nIf you need to copy set from one CDS to another, make a shallow copy by\ncalling dict: s1.data = dict(s2.data)\n'

Reply:
Ahh this just made me realize why the bokeh tutorial is not working for Scott :-D